spec/requests/posts_spec.rb

```diff
@@ -1,7 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] Kindly add a test to check if the response body includes correct placeholder text as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20the%20response%20body%20includes%20correct%20placeholder%20text.) for the `post#index` and `post#show`.